README.md

```diff
@@ -19,7 +19,7 @@ const urlToOptions = require('url-to-options');
 const url = new URL('http://user:pass@hostname:8080/');
 
 const opts = urlToOptions(url);
-//-> { auth:'user:pass', port:8080, … }
+//-> { auth:'user:pass', port:8080, &#x2026; }
```

Comment:
This is odd

Reply:
Hmm.. I think my editor messed it up due to the character being a bit more special, I'll change it back